src/main/com/yetanalytics/datasim/input/alignments.clj

```diff
@@ -81,7 +81,7 @@
   (read-key-fn [this k]
     (keyword nil (name k)))
   (read-body-fn [this json-result]
-    (map->Alignments {:alignment-vector json-result}))
+    (map->Alignments {:alignment-vector (into [] json-result)}))
```

Comment:
Something I noticed is that you tend to use `into []` over `vec`, even though the latter is seemingly more obvious. Is it for efficiency purposes? (We might've talked about this before and I forgot about it.)

Reply:
Efficiency probably doesn't apply here to a great extent, but you can see a great breakdown of some work on `vec` here https://insideclojure.org/2015/01/07/vec-perf/ . I prefer it because it (to me) seems more explicit about the realization of the input collection, as a vector. In the grand scheme of things, this happens once on init and isn't gonna have a big impact either way.

Reply:
In terms of explicitly I take the opposite stance, but to each their own :)

Reply:
it looks too much like a "cast" to me, which it is very much not under the hood. Dunno, we should bench it sometime.